.gitignore

```diff
@@ -1,2 +1,3 @@
 build/
 .python-version
+.idea/
```

Comment:
what is this?

Reply:
I'm not the PR author, but I recognize this as the [.idea folder](https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-) created by [CLion](https://www.jetbrains.com/clion/) and other JetBrains IDEs